docs/Actions.md

```diff
@@ -1063,6 +1073,20 @@ puts release['name']
 
 To get a list of all available values run `fastlane action get_github_release`.
 
+### import_from_git
```

Comment:
We could also add the `import` command here, which just imports the `Fastfile` without any cloning.